src/components/files-directory-link.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Scheduler } from '../handler';
+import { JupyterFrontEnd } from '@jupyterlab/application';
+import { useTranslator } from '../hooks';
+import { Link } from '@mui/material';
+
+export function FilesDirectoryLink(props: {
+  jobFile: Scheduler.IJobFile;
+  app: JupyterFrontEnd;
+}): JSX.Element | null {
+  const trans = useTranslator('jupyterlab');
+  return (
+    <Link
+      href={`/lab/tree/${props.jobFile.file_path}`}
+      title={trans.__('Open output directory with files')}
```

Comment:
This title doesn't seem clear enough. Would "Open output directory in file browser" be clearer, to emphasize that clicking this link would cause a change in the file browser, not in the main work area?